src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -40,7 +39,51 @@ static void Main()
         Console.Write(rn);
     }
 }";
-            CompileAndVerify(source: source, expectedOutput: "TwoZero");
+            CompileAndVerify(source: source, expectedOutput: "TwoZero")
+                .VerifyOperationTree(@"
+Test.Main()
+  IBlockStatement (8 statements, 6 locals) (OperationKind.BlockStatement)
+    Local_1: e (Type: E)
+    Local_2: x (Type: int)
+    Local_3: r (Type: E)
+    Local_4: en (Type: E?)
+    Local_5: xn (Type: int?)
+    Local_6: rn (Type: E?)
```

Comment:
When the test fails, we dump (1) the Assert.Equal failure message which shows the line/char with the first mismatch AND (2) The entire actual operation tree dump, so the user can copy it to a separate buffer and use their preferred diffing tool to compare actual and expected output.